lib/js/components/TextGroup/TextGroup.stories.ts

```diff
@@ -0,0 +1,111 @@
+import { Args, Meta, StoryFn } from '@storybook/vue3';
+import DsDivider from '../Divider/Divider.vue';
+import DsTextGroup from './TextGroup.vue';
+import {
+	TEXT_GROUP_COLORS,
+	TEXT_GROUP_LOADING_SIZES,
+	TEXT_GROUP_SIZES,
+	TEXT_GROUP_STATES,
+} from './TextGroup.consts';
+
+export default {
+	title: 'Components/TextGroup',
+	component: DsTextGroup,
+} as Meta<typeof DsTextGroup>;
+
+const StoryTemplate: StoryFn<typeof DsTextGroup> = (args) => ({
+	components: { DsDivider, DsTextGroup },
+	setup() {
+		return { ...args };
+	},
+	template: `
+		<div style="height: 200px; width: 280px;">
```

Comment:
showing this component on fixed width container makes it harder to understand how does it look like on big screens. Is it something UX wanted?